packages/devtools_app/lib/src/shared/console/widgets/description.dart

```diff
@@ -293,7 +295,21 @@ class DiagnosticsNodeDescription extends StatelessWidget {
       final propertyType = diagnosticLocal.propertyType;
       final properties = diagnosticLocal.valuePropertiesJson;
 
-      if (name?.isNotEmpty == true && diagnosticLocal.showName) {
+      // Show the "default" label if this is the property is the default value:
+      if (diagnosticLocal.level == DiagnosticLevel.fine &&
+          diagnosticLocal.hasDefaultValue) {
+        children.add(
+          Padding(
+            padding: const EdgeInsets.only(right: denseSpacing),
+            child: DefaultValueLabel(
+              colorScheme: colorScheme,
+              textStyle: textStyle,
```

Comment:
don't pass these values in, but instead, look them up in the build method for DefaultValueLabel

Reply:
Done!